nvflare/app_opt/xgboost/histogram_based_v2/sec/partial_he/util.py

```diff
@@ -16,8 +16,14 @@
 from base64 import urlsafe_b64decode, urlsafe_b64encode
 from binascii import hexlify, unhexlify
 
-import ipcl_python
-from ipcl_python import PaillierEncryptedNumber as EncryptedNumber
+try:
+    import ipcl_python
+    from ipcl_python import PaillierEncryptedNumber as EncryptedNumber
+    from ipcl_python.ipcl_python import BNUtils, ipclCipherText
+
+    ipcl_imported = True
+except Exception:
+    ipcl_improted = False
```

Comment:
Typo: ipcl_improted => ipcl_imported
But what is this for? If not imported successfully, then what?